### ISSUE TITLE
RC Test: Friction 1.4.0-rc.2 for Keyboard Navigation

### ISSUE THREAD

**Issue by zepumph** [phetsims/QA#75]
@emily-phet @terracoda @mbarlow12 @jessegreenberg  Friction 1.4.0-rc.2 is ready for RC testing. RC.1 was quickly updated to support accessibility features by default, without the need for a query flag.

**[Link to sim](https://www.colorado.edu/physics/phet/dev/html/friction/1.4.0-rc.2/friction_en.html)**

**[Link to iFrame](https://www.colorado.edu/physics/phet/dev/html/friction/1.4.0-rc.2/friction_en-iframe.html)**

**[Test Matrix](https://docs.google.com/spreadsheets/d/1cnoPcw-DPCOpy4fTgoDPQhb0_sX2Z26J3W66Z3IQxtw/edit#gid=917763661)**

**Issues to Verify**

No issues to verify, but please hammer on keyboard navigation as this is only the 4th simulation that has been tested with it.

**Please also verify**
- [x] stringTest=double (all strings doubled)
- [x] stringTest=long (exceptionally long strings)
- [x] stringTest=X (short strings)
- [x] stringTest=rtl (right-to-left)
- [x] stringTest=xss (test passes if sim does not redirect, OK if sim crashes or fails to fully start)
- [x] showPointerAreas (touchArea=red, mouseArea=blue)
- [x] Full screen test
- [x] Screenshot test
- [x] Does AR need to be notified for any LOL updates?

If any new issues are found, please note them in https://github.com/phetsims/friction/issues and reference this issue.  

@ariel-phet please assign and prioritize

**Comment by phet-steele** [comment]
@jessegreenberg @zepumph, testing is done here.

**Comment by phet-steele** [comment]
Next rc up.
